OneLauncher/Settings.py

```diff
@@ -60,7 +60,7 @@ def LoadSettings(self, useGame=None):
         self.winePrefix = self.settingsDir + "wine/prefix"
         self.builtInPrefixEnabled = True
         self.gameDir = ""
-        self.x64ClientEnabled = True
+        self.client = ""
```

Comment:
"Client" and "x64Client" should always agree, so you can default self.client to "WIN64" at its definition and remove the check for self.client == "". You also need to have self.client = "WIN32" when "x64Client" is false.